docs/application/dotnet/guides/nui/grid-layout.md

```diff
@@ -0,0 +1,29 @@
+# Grid Layout
+
+`GridLayout` is a grid box for the two dimensional layout. It constraints the x and y position, width, and height of the child actors.
+
+Positioning children in a grid form, the cells are of uniform size based on the first child added to the parent View.
+
+The number of **columns** can be specified and the rows will automatically increase to hold the children. Once the available space is used up, further rows will not be added.
```

Comment:
If so, only the **columns** is the controllable value to the user?

Also it looks like if a user tries to add more children than the available space, the left children will just ignored or no response to the adding trial would be happen. Am I right?

Reply:
@rishu-samsung Thanks for the rephrase. However I'm thinking of the second sentence.

After the available space is used, the remaining rows become invisible by default.

I think the original meaning is "When every available space is used up, rows added after that time is invisible by default option.". With the word "remaining", to me the meaning is slightly changed as it sounds like the existing rows outside of the available space became invisible. Would you please check again?

Reply:
@mijin85cho 
Thank you for the explanation. Can we rephrase to:
You can set the number of **columns**, however, the rows automatically increase to hold the children. By default, the rows added after the available space is used become invisible.

Reply:
@rishu-samsung I think this modification is better.

Question to "after the available space is used "
Isn't the "after the available space is used up" intended, to get the meaning of "you exhaust every available space"? Is there any reason you removed "up" from "used up"?

Reply:
@mijin85cho,

The reason SRI-B removed "up" from "used up" is because, "used" and "used up" provide the same meaning. However, to ensure minimalist writing and also ensure that the meaning is not changed, we suggested "used". Suggestion:

You can set the number of columns, however the rows automatically increase to hold the children. After the available space is used, the remaining rows become invisible by default.

Reply:
Okay I rephrase the sentence to the last organized one.